src/components/bc-balance.ts

```diff
@@ -30,13 +31,21 @@ export class Balance extends withTwind()(BitcoinConnectElement) {
 
   override render() {
     // TODO: if balance is still loading, show skeleton loader
-    return html` <span
-      class="font-medium font-sans mr-2 flex justify-center items-center gap-0.5 ${classes[
-        'text-brand-mixed'
-      ]}"
-    >
-      <span class="font-mono">${this._balance || 0} </span>&nbsp;sats</span
-    >`;
+    const numericBalance = parseFloat((this._balance || '0').replace(/,/g, ''));
```

Comment:
this is not needed - the `_loadBalance` function purposefully converts the balance to a string. I think we can add fiat support there instead, if the currency is not BTC.